operators/basic_network/metadata.json

```diff
@@ -1,24 +1,24 @@
 {
 	"operator": {
-		"name": "basic_network",
+		"name": "basic_networking",
```

Comment:
Is the name "basic_network" or "basic_networking"?  I see in basic_networking_ping's metatdata.json file it's referenced as "basic_network" and looking at advanced_network operator, you also named that one without the "ing"

Reply:
I'll fix this to make it uniform on both

Reply:
@cliffburdick any updates on this one?  I think we just need to make this small change here, and possibly add a guard to prevent the app from crashing when user forgets to pass the config yaml file.

Reply:
I'll try to get to it today. I switched to work on ANO

Reply:
@wendell-hom this is fixed.